cvat/apps/dataset_manager/tests/test_rest_api_formats.py

```diff
@@ -1045,6 +1045,47 @@ def test_api_v2_tasks_annotations_dump_and_upload_many_jobs_with_datumaro(self):
                     data_from_task_after_upload = self._get_data_from_task(task_id, include_images)
                     compare_datasets(self, data_from_task_before_upload, data_from_task_after_upload)
 
+    def test_api_v2_tasks_annotations_dump_and_upload_with_datumaro_outside_true(self):
```

Comment:
Please move the test into `/tests/python`. We're trying to move away from the django-based tests, so all the new tests should be added in the "new" tests, with some exceptions for server internals checks.